doc/conf.py

```diff
@@ -27,7 +26,7 @@
 exclude_patterns = ['_build']
 
 project = 'Sphinx'
-copyright = f'2007-{time.strftime("%Y")}, the Sphinx developers'
+copyright = '2007-%Y, the Sphinx developers'
```

Comment:
Although I think we should support this templating for other projects, I have a strong preference for using constant/static copyright notices where possible (although I fully admit that adding supporting tooling to detect cases where they're out-of-date could be useful).

Given that, and the fact that we update most of the copyright notices throughout the codebase manually each year, I wonder whether we should do the same here.

```suggestion
copyright = '2007-2024, the Sphinx developers'
```